### ISSUE TITLE
Add option for outputting CloudFormation instead of Terraform

### ISSUE THREAD

**Issue by mcbanderson** [cloudmitigator/reflex-cli#24]

**Comment by rot26** [comment]
I don't think this is something we have determined that we would want to support unless you have a magic tf / cfn converter.
Closing.  Reopen if there is still interest.

**Comment by mcbanderson** [comment]
This is a feature that has been requested, so I'm reopening. We should be able to create a separate set of CFN templates which we output instead of TF, so we wouldn't have to worry about generic TF -> CFN conversion.
